### DISCUSSION TITLE
How can I render html in a message?

### DISCUSSION THREAD

**Discussion by sawa-ko** [nuxt-modules/i18n#1818] [Q&A]
I am trying with `a <span>s</span>` but the page only shows the message as text and not as html.

![image](https://user-images.githubusercontent.com/56084970/214371877-e04c52c4-73d2-499a-9595-54c04606993f.png)

**Comment by xibman** [comment]
@kaname-png  see : https://vue-i18n.intlify.dev/guide/essentials/syntax.html#html-message
